src/it/scala/temple/containers/GolangSpec.scala

```diff
@@ -25,12 +26,21 @@ abstract class GolangSpec extends DockerShell2HttpService(8081) with DockerTestK
   }
 
   def validateAll(files: Files, entryFile: File): String = {
-    val json = files.map { case (file, contents) => (file.folder + "/" + file.filename, contents) }.asJson.toString()
-    Http(golangVerifyUrl)
+    val json = files.asJson.toString()
+    val errors = Http(golangVerifyUrl)
       .params(Map("src" -> json, "root" -> entryFile.folder, "entrypoint" -> entryFile.filename))
-      .timeout(connTimeoutMs = 1000, readTimeoutMs = 30000)
+      .timeout(connTimeoutMs = 1000, readTimeoutMs = 60000)
       .asString
       .body
+    if (errors.nonEmpty) {
+      for ((file, content) <- files.to(SortedMap)) {
+        System.err.println(s"$file:")
+        for ((line, i) <- content.linesIterator.zipWithIndex) {
+          System.err.println(s"${i + 1}:".padTo(6, ' ') + line)
```

Comment:
6? 🧙

Reply:
surely no more than 4 digits for the row number, usually 3, then a colon, then a space. Would you rather I commented it?

Reply:
I don’t think I would have guessed that explanation, so yes please 👍